NeedABreak/SettingsWindow.xaml

```diff
@@ -156,6 +156,16 @@
 				Click="CloseButton_Click">
             <TextBlock Text="{utils:TextResource fermer}"
 					   Foreground="White" />
-		</Button>
-	</Grid>
+        </Button>
+
+        <Button Grid.Row="3"
+				BorderThickness="0"
+				FontSize="20"
+				Height="50"
+				Margin="0,25,654,25"
+				Click="LockNow" HorizontalAlignment="Right" Width="116">
+            <TextBlock Text="Lock Now"
```

Comment:
Please use utils:TextResource markup extension instead of hard-coded text. You don't have to make translations, just add a new string in resource.resx file.

Reply:
Ah right! no problem. I'll do that now.